bundles/core/src/test/resources/title/exporter-title-jcr-title-link-v2.json

```diff
@@ -11,7 +11,12 @@
     },
     "link": {
         "url": "https://www.adobe.com",
-        "valid": true
+        "valid": true,
+        "htmlAttributes": {
+            "href":"https://www.adobe.com",
```

Comment:
With this change, the link URL appears three times (without counting the data layer) in the JSON:
```
{
    "id": "title-6032c7e80a",
    "linkDisabled": false,
    "linkURL": "https://www.adobe.com",
    "dataLayer": {
        "title-6032c7e80a": {
            "repo:modifyDate": "2016-01-13T15:14:51Z",
            "xdm:linkURL": "https://www.adobe.com",
            "@type": "core/wcm/components/title/v2/title"
        }
    },
    "link": {
        "url": "https://www.adobe.com",
        "valid": true,
        "htmlAttributes": {
            "href":"https://www.adobe.com",
            "aria-label":"World",
            "title":"World title"
        }
    },
    ":type": "core/wcm/components/title/v2/title"
}
```
I'd prefer to get rid of the `link` JSON object, as it's weird anyway to have HTML attributes in JSON.

Reply:
I would suggest following JSON:

**Title v2**
```
{
    "id": "title-6032c7e80a",
    "linkDisabled": false,
    "linkURL": "https://www.adobe.com",
    "dataLayer": {
        "title-6032c7e80a": {
            "repo:modifyDate": "2016-01-13T15:14:51Z",
            "xdm:linkURL": "https://www.adobe.com",
            "@type": "core/wcm/components/title/v2/title"
        }
    },
    ":type": "core/wcm/components/title/v2/title"
}
```

**Title v3**
```
{
    "id": "title-6032c7e80a",
    "linkDisabled": false,
    "dataLayer": {
        "title-6032c7e80a": {
            "repo:modifyDate": "2016-01-13T15:14:51Z",
            "xdm:linkURL": "https://www.adobe.com",
            "@type": "core/wcm/components/title/v2/title"
        }
    },
    "link": {
        "url": "https://www.adobe.com",
        "valid": true,
        "htmlAttributes": {
            "aria-label":"World",
            "title":"World title"
        }
    },
    ":type": "core/wcm/components/title/v2/title"
}
```
@gabrielwalt WDYT?